VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.ts

```diff
@@ -252,6 +252,11 @@ export class SelfTestComponent implements OnInit, OnDestroy {
             participant: this.selfTestParticipantId
         });
         this.disconnect();
+
+        this.userMediaStreamService.activeMicrophoneStream$
+            .pipe(takeUntil(this.destroyedSubject))
```

Comment:
If this is the action from the replay video button. It should be a take one instead of take until destroyed.